fe/fe-core/src/main/java/com/starrocks/common/Config.java

```diff
@@ -1402,4 +1402,7 @@ public class Config extends ConfigBase {
      */
     @ConfField(mutable = true)
     public static int default_bucket_num = 10;
+
+    @ConfField
```

Comment:
mutable

Reply:
We couldn't make enable_dict_optimize_routine_load mutable, because we disable optimize when we create routine load or replay the the log. So it's meanless to make make the config mutable

Reply:
If we disable optimize when we create routine load, why the title called "support routine load in lowcardinality optimization"